classes/Cluster.py

```diff
@@ -120,3 +120,6 @@ def __repr__(self):
             f"Cluster {self.id}: {len(self.scooters)} scooters, current state: {self.get_current_state()},"
             f" ideal state: {self.ideal_state}"
         )
+
+    def prob_of_scooter_usage(self):
+        return len(self.get_available_scooters()) / self.ideal_state
```

Comment:
Do we want to give a negative reward if there are more scooters available than ideal state...?

Reply:
`1-len(self.get_available_scooters()) / self.ideal_state"